students/junsikchoi/westagram/urls.py

```diff
@@ -1,18 +1,3 @@
-"""westagram URL Configuration
+from django.urls import path, include
 
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.2/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
-from django.urls import path
-
-urlpatterns = []
+urlpatterns = [path("users", include("users.urls"))]
```

Comment:
오히려 여기는 너무 빡빡하게 해서 가독성이 떨어지네요.
```suggestion
urlpatterns = [
    path("users", include("users.urls"))
]
```

Reply:
음.. 주관적으로 말씀하셔서 조금 헷갈립니다. 기준 글자수를 알려주시면 정확히 설정해서 다시 포매팅 후 올리겠습니다.